providers/firetext/src/lib/firetext.provider.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+
+if (!globalThis.fetch) {
+  // eslint-disable-next-line global-require
+  globalThis.fetch = require('node-fetch');
+}
```

Comment:
What is the reason behind setting Fetch's Node implementation in the global scope? What do we need this for? Also I can see it is not added to the list of dependencies.

Reply:
I've taken this approach as this was used by the `gupshup` provider. I wasn't sure if there was a reason, e.g. needing to run in the browser, that I didn't know about. This also avoids importing fetch for newer versions of Node with fetch included.

Happy to change to a standard import not on the global scope if preferred.

Reply:
Regarding dependencies, `node-fetch` has been added to the `package.json` for this package. Is there somewhere else it needs to be added as well?